project.mml

```diff
@@ -2195,6 +2195,7 @@ Layer:
               'barrier_' || CASE WHEN barrier IN ('bollard', 'gate', 'lift_gate', 'swing_gate', 'block', 'log', 'cattle_grid', 'stile', 'motorcycle_barrier', 'cycle_barrier', 'full-height_turnstile', 'turnstile', 'kissing_gate') THEN barrier ELSE NULL END
             )  AS feature,
             access,
+            tags->'parking' as parking,
```

Comment:
You also need to add this to the `amenity-points` layer, since `[parking='underground']` is used in line 2927 of `amenity-points.mss` for the text, but parking features are no longer in `amenity-low-priority`. Right now I get the error `column "parking" does not exist`.